tools/faucet/cmd/cli.go

```diff
@@ -41,15 +45,25 @@ func parseCLIArgs() *faucet.Config {
 	faucetPK := flag.String(faucetPKName, faucetPKDefault, faucetPKUsage)
 	jwtSecret := flag.String(jwtSecretName, jwtSecretDefault, jwtSecretUsage)
 	serverPort := flag.Int(serverPortName, serverPortDefault, serverPortUsage)
+	defaultAmount := flag.Float64(defaultAmountName, defaultAmountDefault, defaultAmountUsage)
 	flag.Parse()
 
 	return &faucet.Config{
-		Port:       *faucetPort,
-		Host:       *nodeHost,
-		HTTPPort:   *nodeHTTPPort,
-		PK:         *faucetPK,
-		JWTSecret:  *jwtSecret,
-		ServerPort: *serverPort,
-		ChainID:    big.NewInt(443), // TODO make this configurable
+		Port:              *faucetPort,
+		Host:              *nodeHost,
+		HTTPPort:          *nodeHTTPPort,
+		PK:                *faucetPK,
+		JWTSecret:         *jwtSecret,
+		ServerPort:        *serverPort,
+		ChainID:           big.NewInt(443), // TODO make this configurable
+		DefaultFundAmount: toWei(defaultAmount),
 	}
 }
+
+func toWei(amount *float64) *big.Int {
+	amtFloat := new(big.Float).SetFloat64(*amount)
+	weiFloat := new(big.Float).Mul(amtFloat, big.NewFloat(1e18))
+	// don't care about the accuracy here, float should have less than 18 decimal places
+	wei, _ := weiFloat.Int(nil)
+	return wei
+}
```

Comment:
Perhaps `gethcommon.ParseEther(ethValue)` is a better alternative.

Reply:
That does sound better but I can't find it. I've changed 1e18 magic number to use geth's `params.Ether` but seems like in geth they mostly just do the multiplication.

Reply:
Could swear there was a ParseEther(string), but had a double look and nothing. params.Ether sounds good.